plugins/module_utils/foreman_helper.py

```diff
@@ -971,10 +962,11 @@ def run(self, **kwargs):
                 self.foreman_params[self.entity_key] = self.foreman_params.pop(updated_key)
 
         params = kwargs.get('params', {})
-        if self.entity_scope:
-            for scope in self.entity_scope:
+        entity_scope = self.foreman_spec['entity'].get('scope')
+        if entity_scope:
+            for scope in entity_scope:
```

Comment:
you could write `for scope in self.foreman_spec['entity'].get('scope', []):` here… but I don't think it's needed

Reply:
Do you think, it's more readable? Because that's the only reason why i would do it.

Reply:
It saves two lines, whether this is more readable depends on the reader, I guess. So probably just leave it as it is.